### ISSUE TITLE
Keybinding to search layouts

### ISSUE THREAD

**Issue by traek728** [Leleat/Tiling-Assistant#241]
**Describe the bug**
The "Search for a Layout" popup will only activate when the cursor is on the far left monitor in my display arrangement when using the keybinding I set to search layouts in advanced settings of the extension.   Also, even when the cursor is on the far left monitor and you execute the keybinding to search layouts, the search popup, while it will show, displays itself on a different monitor entirely (2 monitors down the line in my display arrangement).

**Steps To Reproduce**

**System Info:**
 - Distro (incl. version):  Fedora 37
 - GNOME Shell version: 43.3
 - Extension version and from where (e. g. EGO, `main` branch...): Latest version listed in gnome extensions site (currently version 39)
 - XOrg/Wayland: Wayland, Nvidia RTX 3070 (proprietary drivers)

**Journalctl logs**
  No relevant log output

**Comment by Leleat** [comment]
I can't seem to reproduce this on a dual monitor setup. The popup appears on the monitor where the pointer is, as it should. The code is for this pretty straigtforward

https://github.com/Leleat/Tiling-Assistant/blob/main/tiling-assistant%40leleat-on-github/src/extension/layoutsManager.js#L354:

```
        const monitor = global.display.get_current_monitor();
        const workArea = activeWs.get_work_area_for_monitor(monitor);
        popup.set_position(workArea.x + workArea.width / 2 - popup.width / 2,
            workArea.y + workArea.height / 2 - popup.height / 2);
```

So no idea what could be causing this or how to fix it...

**Comment by Leleat** [comment]
Could you please open looking glass (press `Alt`+`F2` then enter `lg`). Then enter the following lines

```
const activeWs = global.workspace_manager.get_active_workspace(); 
return [
   activeWs.get_work_area_all_monitors().x,
    activeWs.get_work_area_all_monitors().y,
    activeWs.get_work_area_all_monitors().width,
    activeWs.get_work_area_all_monitors().height
];
```

and paste the return value here.

**Comment by Leleat** [comment]
> Could you please open looking glass (press Alt+F2 then enter lg). Then enter the following lines

No need to do this anymore. I am pretty sure that was the issue. So I fixed in https://github.com/Leleat/Tiling-Assistant/pull/247. Do you want to try out the PR before I merge it?

**Comment by traek728** [comment]
Resolved in #247
